test/unit/resources/legacy_gelamtest.js

```diff
@@ -0,0 +1,351 @@
+/**
+ * LegacyGelamTest: Supporting actor-style tests after it was cool.
```

Comment:
Thanks much for the excellent block comment here.  And major props to you for distilling the testdriver/testcontext/log spaghetti soup into something so concise.  I'm sure you spent a lot of time on this file... or just cursing ;)